README.md

```diff
@@ -20,34 +22,40 @@ determine them. The label can be applied with:
 ```bash
 kubectl label nodes <node-name> audit=target
 ```
-Connect to the control Node and copy `audit-policy.yaml` and `audit-config.yaml`
-to `/etc/kubernetes/manifests`. Run the following commands to create the
-directory the repository will be stored on.
+Run the following command to create the directory the repository will be stored
+on:
```

Comment:
s/stored on/stored in

I think you should specify that this directory should be created on the Node which has been labelled
I wonder if this step is actually necessary, won't the directory be created automatically if needed?

Reply:
At the moment it doesn't, but I'll change the deployment to reflect that.

Reply:
I don't think I can add a configuration to the manifest to create the PV directory if it doesn't exist.

Reply:
I thought the PV would take care of it by default (or rather kubelet when handling the PV). If not the case, please ignore.

Reply:
You should still specify that the directory needs to be manually created on the Node that was labelled (i.e. the Node on which the PV will be created).